mkdocs.yml

```diff
@@ -10,6 +10,7 @@ nav:
     - Deploying your first model: 'getting-started.md'
   - Concepts:
     - Workflows: 'workflows.md'
+    - Data Version Control: 'data_version_control.md'
```

Comment:
Not sure if `Concepts` is the best location for this? I think either in `Inside Matcha` or create a new section something along the lines of `Tools` either separate or within the `Inside Matcha` section. What do you think?

Reply:
I'm not sure either. Concepts seemed a good fit at the time of writing, but a new section for how to interact with provisioned resources for other tools seems like a great idea. I'll do that.